Formula/ooniprobe.rb

```diff
@@ -166,16 +166,57 @@ def install
     (var/"lib/ooni").mkpath
   end
 
-  def post_install
-    system bin/"ooniresources"
-    system bin/"oonideckgen", "-o", "#{HOMEBREW_PREFIX}/share/ooni/decks/"
```

Comment:
Yes that makes sense. In fact `brew audit --strict --online ooniprobe` hinted at this :

```
  * `caveats method` (line 215) should be put before `plist block` (line 169)
```

and that was why I moved the `caveats`, but didn't move the `post_install`. Anyways will fix.